product_variant_inactive/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of Product Variants
```

Comment:
This module extends the functionality of product variants
adding 2 buttons to activate and deactivate them.